services/api/src/typeDefs.js

```diff
@@ -928,6 +928,11 @@ const typeDefs = gql`
     environments: [Environment]
   }
 
+  type EventType {
+    name: String
+    type: String
```

Comment:
Doesn't it make sense to, at this stage, actually return an array of types in case events have multiple categories.